pixi.toml

```diff
@@ -363,11 +363,15 @@ py-build-notebook = { cmd = "pip install -e rerun_notebook", depends-on = [
   "js-build-base",
 ] }
 
+# Build an installable wheel. This needs `RERUN_ALLOW_MISSING_BIN=1` because the `rerun` binary is not part of the
```

Comment:
your doc string is now outdated ;-)

Reply:
will likely fail on windows just because the file path is a different one. But I can check tonight, wouldn't even be surprised if pixi handles this